pkg/advisor/utils.go

```diff
@@ -136,13 +136,13 @@ func findReplicaset(replicasets *appsv1.ReplicaSetList, dep appsv1.Deployment) (
 	return nil, fmt.Errorf("could not find replicaset for deployment '%s' gen '%v'", dep.Name, generation)
 }
 
-func makePrometheusClientForCluster() (*promClient, error) {
+func makePrometheusClientForCluster(namespace string) (*promClient, error) {
 	config, _, err := findConfig()
 	if err != nil {
 		return nil, err
 	}
 
-	promurl := fmt.Sprintf("%s%s", config.Host, promOperatorClusterURL)
+	promurl := fmt.Sprintf(promOperatorClusterURL, config.Host, namespace)
```

Comment:
this is not correct, it should be config.Host which should equal to kubernetes api url